### ISSUE TITLE
TypeError: Path must be a string. Received undefined

### ISSUE THREAD

**Issue by Turbo87** [ember-cli/ember-cli-htmlbars-inline-precompile#155]
After updating to v2.0.0 I've started seeing the following errors on CI:

```
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.resolve (path.js:1152:7)
    at Class.templateCompilerPath (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli-htmlbars-inline-precompile/index.js:129:17)
    at Class.included (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli-htmlbars-inline-precompile/index.js:60:39)
    at Class.superWrapper [as included] (/home/travis/build/emberjs/ember-mocha/node_modules/core-object/lib/assign-properties.js:34:20)
    at project.addons.forEach.addon (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli/lib/broccoli/ember-app.js:636:15)
    at Array.forEach (native)
    at EmberAddon._notifyAddonIncluded (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli/lib/broccoli/ember-app.js:634:25)
    at EmberAddon.EmberApp (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli/lib/broccoli/ember-app.js:146:10)
    at EmberAddon (/home/travis/build/emberjs/ember-mocha/node_modules/ember-cli/lib/broccoli/ember-addon.js:66:5)
```

(see https://github.com/emberjs/ember-mocha/pull/269 and https://travis-ci.org/emberjs/ember-mocha/jobs/466133058)

**Comment by rwjblue** [comment]
Tldr; 2.0 dropped support for non-npm versions of Ember so in order to test Ember < 2.12 you have to use 1.x.

We should absolutely make this a better error message...

**Comment by rwjblue** [comment]
FYI - This was introduced by https://github.com/emberjs/ember-test-helpers/pull/481.

**Comment by rwjblue** [comment]
And bower support was dropped in https://github.com/ember-cli/ember-cli-htmlbars-inline-precompile/pull/126
